packages/suite-desktop/scripts/gnupg-sign.sh

```diff
@@ -0,0 +1,10 @@
+if [ -z "$GPG_CSC_USERID" -o -z "$GPG_CSC_LINK" -o -z "$GPG_CSC_KEY_PASSWORD" ]; then
+  echo "GPG sign skipped"
+  exit 0
+fi
+
+gpg --batch --import "$GPG_CSC_LINK"
+
+for f in packages/suite-desktop/build-electron/*.{AppImage,dmg,exe} ; do
```

Comment:
Does it make sense to sign also dmg and exe?

Reply:
Yes, we do it already and the signatures are linked from the landing page. It certainly does not hurt.

Reply:
Ok, true :)